release/mixer.sh

```diff
@@ -78,20 +78,26 @@ build_update() {
 }
 
 build_deltas() {
+    local mix_format="${1}"
+
     section "Deltas"
 
-    log "Building delta packs..." 1
-    if (( NUM_DELTA_BUILDS > 0 )); then
-        sudo_mixer_cmd build delta-packs --previous-versions "${NUM_DELTA_BUILDS}"
+    local first_file="update/www/versions/format${mix_format}/first"
```

Comment:
We should bring that to the attention of the mixer team and see if it can be created by mixer (even that through an optional flag so it is not enforced to all mixes).

Reply:
I'll check with them.  But I think that because we've copied afb.sh into clr-distro-factory, we take ownership of the format bump process, which would also include the first file.  Maybe motivation to make automatic format bumps better?

Reply:
When I think mixer should take care of it, Im not saying the afb. The tools should dump that file there if it is the first of a format IMHO